.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
nit: this is a good candidate for your personal `~/.gitignore` file, rather than here. ;)

Reply:
TIL. Thanks for commenting about that!

Reply:
I removed that through https://github.com/samuelallan72/rusty-charm-framework/pull/1/commits/e5711a9ada202c40458317041c87378be5132f79.